src/containers/collectibles/components/types.ts

```diff
@@ -0,0 +1,27 @@
+export type CollectiblesMetadata = {
+  [key: string]: object
+  order: string[]
+}
+
+export enum CollectibleType {
+  IMAGE = 'IMAGE',
+  VIDEO = 'VIDEO'
+}
+
+export type Collectible = {
+  id: string
+  name: string | null
```

Comment:
Michael made a nice `Nullable<TYPE>` wrapper that we try to use instead of `TYPE | null`

Reply:
cool